src/components/Search/Search.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MoviesList } from '../MoviesList';
+
+export class Search extends React.Component {
+  state = {
+    query: '',
+  }
+
+  render() {
+    const { moviesList } = this.props;
+    const { query } = this.state;
+    let filteredMoveisList = [];
+
+    if (query) {
+      const searchPhrase = query.toLocaleLowerCase();
+
+      filteredMoveisList = moviesList.filter(({ title, description }) => (
+        title.toLocaleLowerCase().includes(searchPhrase)
+            || description.toLocaleLowerCase().includes(searchPhrase)
+      ));
+    }
+
+    return (
+      <>
+        <div className="box">
+          <div className="field">
+            <label htmlFor="search-query" className="label">
+              Search movie
+            </label>
+
+            <div className="control">
+              <input
+                type="text"
+                id="search-query"
+                className="input"
+                placeholder="Type search word"
+                value={this.state.query}
+                onChange={(event) => {
+                  this.setState({ query: event.target.value });
```

Comment:
```suggestion
                  this.setState({ query: event.target.value.toLowerCase() });
```
Мы можем прямо тут приводить к нижнему регистру, тогда не будет потребности в доп. переменной вверху.

Reply:
I need to put query.toLocaleLowerCase() in separate variable searchPhrase in file Search.jsx to calculate it just ones and use it further. In case In case I put it in onChange method I will call .toLocaleLowerCase() every time onChange method will be invoked so isn't optimal